Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -125,24 +125,31 @@
 
 - type: entity
   parent: ClothingEyesBase
-  id: ClothingEyesGlassesSunglasses
-  name: sun glasses
-  description: A pair of black sunglasses.
+  id: ClothingEyesGlassesCheapSunglasses
+  name: cheap sun glasses
+  description: A pair of black sunglasses, does not protect against flashing.
```

Comment:
the description is kinda blatant. Maybe just say it's bad at blocking light or something similar.

Reply:
Okay